python/ray/autoscaler/kubernetes/operator_configs/operator_cluster_scoped.yaml

```diff
@@ -13,6 +13,9 @@ rules:
 - apiGroups: ["", "cluster.ray.io"]
   resources: ["rayclusters", "rayclusters/finalizers", "rayclusters/status", "pods", "pods/exec", "services"]
   verbs: ["get", "watch", "list", "create", "delete", "patch", "update"]
+- apiGroups: [""]
+  resources: [events]
+  verbs: [create]
```

Comment:
just curious, when does the operator create an event

Reply:
I believe each time it finishes a status handler (create, update, delete) it emits an event as a result of modifying the object metadata. Without this, I was seeing errors.

Reply:
Makes sense, thanks!